src/extensions/sticky-header/README.md

```diff
@@ -0,0 +1,24 @@
+# Table sticky-header
+
+This is an extension for [Bootstrap table](http://github.com/wenzhixin/bootstrap-table) module which provides a sticky header for the table when scrolling. </br>
+You must include the bootstrap-table-sticky-header.css file in order to get the appropriate style
+
+## Usage
+
+```html
+<script src="extensions/group-by/bootstrap-table-sticky-header.js"></script>
```

Comment:
Hi @wenzhixin,

This is my first pull request.

For adding an example, I can't git push to master branch in: https://github.com/wenzhixin/bootstrap-table-examples/tree/master/extensions repo

I've committed to my local repo